packages/bundle/src/createDirectLine.js

```diff
@@ -1,12 +1,13 @@
 import { DirectLine } from 'botframework-directlinejs';
 
-export default function ({ domain, fetch, secret, token, webSocket }) {
+export default function ({ domain, fetch, secret, token, webSocket, conversationId }) {
```

Comment:
Please sort the keys alphabetically :)

Reply:
Sorted the keys alphabetically in Directline js constructor  and also updated CHANGELOG.md